packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -16,56 +19,62 @@ const StyledBox = styled(Box)`
   position: relative;
 `
 
-const StyledHeading = styled(Heading)`
-  text-shadow: 0 2px 4px rgba(0,0,0,0.5);
-`
-
-const StyledAnchor = styled(Anchor)`
-  border-bottom: 3px solid transparent;
-  &:focus,
-  &:hover {
-    text-decoration: none;
-    border-color: white;
-  }
-`
-
 function ProjectHeader (props) {
-  const { className, href, title } = props
-
-  const Title = () => (
-    <StyledHeading color='white' margin='none' size='small'>
-      {title}
-    </StyledHeading>
-  )
+  const { className, navLinks, projectHomeLink, size, title } = props
 
   return (
-    <StyledBox className={className}>
+    <StyledBox>
```

Comment:
Why was className removed here?

Reply:
I don't think it made any difference - I didn't see any class warnings as a result.